npm/src/index.ts

```diff
@@ -38,6 +38,8 @@ export const renderModules = async () => {
   const fileChecksContainer: HTMLDivElement | null = document.querySelector(
     ".file-check-progress"
   )
+  const timeoutDialog: HTMLDialogElement | null =
```

Comment:
The `HTMLDialogElement` is showing its deprecated. Having a quick look, it appears this is a bug with Typescript: https://github.com/microsoft/TypeScript/issues/48267 Could you double check that this is the case.

Reply:
The HTMLDialogElement isn't showing as deprecated for me. It does say experimental, but from what I have gathered many browsers do support this. I have also tested it out with various browsers myself.